pkgs/by-name/do/dooit/package.nix

```diff
@@ -33,7 +34,14 @@ python3.pkgs.buildPythonApplication rec {
     pyperclip
     python-dateutil
     pyyaml
-    textual
+    (python3.pkgs.textual.overrideAttrs (oldAttrs: rec {
+      version = "0.47.0";
+      src = fetchPypi {
+        pname = "textual";
+        version = "0.47.0";
+        sha256 = "sha256-YTkSBBqoc55O9yi7KYttAPK/4/5SisZLwMzAb0/2VuA=";
+      };
+    }))
```

Comment:
```suggestion
 (textual.overridePythonAttrs (oldAttrs: {
      version = "0.47.1";
      src = fetchFromGitHub {
        owner = "Textualize";
        repo = "textual";
        rev = "refs/tags/v0.47.1";
        hash = "sha256-RFaZKQ+0o6ZvfZxx95a1FjSHVJ0VOIAfzkdxYQXYBKU=";
      };
      disabledTests = [
        "test_tracked_slugs"
        "test_textual_env_var"
        "test_register_language"
        "test_register_language_existing_language"
      ];
    }))
```

Reply:
This at least gets it building. But, the contents of the tests failing seemed like it would indicate another issue going on. Try this and see if it fixes the issue you were seeing.

Reply:
Yes I checked it now. Its failing for some reason, it was working fine a few days ago. idk. Im new to nix myself so maybe I didn't understand something but thank you, I'll test again